README.md

```diff
@@ -101,43 +89,17 @@ localhost:4000 by default.
 
 ### Editing Docs
 
-Use this procedure to add or modify the CircleCI docs and make a pull request from your local clone.
-
-1. To see the list of the Markdown files for CircleCI v2.0, change to the `jekyll/_cci2` directory and type `ls`.
-
-`cd jekyll/_cci2`
-
-`ls`
-
-See the `CONTRIBUTING.md` file in this repository for documentation style tips. 
+All docs live in folders named after the version of CircleCI. The only 2 you need to worry about are `jekyll/_cci1` and `jekyll/_cci2`, for CircleCI Classic and CircleCI 2.0, respectively.
 
-2. Add or modify the Markdown files and rebuild the site to check your work. 
+Add or modify the Markdown files in these directories according to the style guide outlined in `CONTRIBUTING.md` in this repository. Rebuild the site when you want to check your work with:
 
 `./jctl rebuild`
 
 For more detailed instructions on using `jctl`, see [Jekyll Controller Commands](#jekyll-controller-commands).
 
-3. Create a branch and switch to it.
-
-`git branch <branch-name>`
-
-`git checkout <branch-name>`
-
-4. Stage and commit your changes.
-
-`git commit -a -m "commit message"`
-
-5. Download GitHub Desktop from <https://desktop.github.com/> if you haven’t already and open it. 
-
-6. To add your local repository, click the + button, select Add, and choose the `<my-github-username>/circleci-docs` folder. 
-
-The `<my-github_username>/circleci-docs` repo appears under GitHub in the left pane.
-
-7. Click `<my-github_username>/circleci-docs` to see the committed changes in your branch.
-
-8. Click the Pull Request button, add a description, and click Send Pull Request to submit your changes for review.
+### Submitting a Pull Request
 
-You will get a notification when a reviewer responds to your request and after the changes are merged.
+Create a pull request by following [GitHub's guide](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
```

Comment:
Is your working assumption that people who don't already understand branching/pushing/PR should not be relying on our summary? I suspect most potential editors DO understand the flow to a PR from a fork, but I'm not clear on the benefit of killing these instructions other than brevity.